app/Repositories/UserRepository.php

```diff
@@ -161,4 +161,53 @@ public function updateUserData($user, $updateData = [])
             return $user;
         });
     }
+
+    public function bulkCreate($request)
+    {
+        $users = $request->get('users');
+        $address_array = $request->get('address_array');
+        $circle_id = $request->route('circle_id');
+        return DB::transaction(function () use ($users, $address_array, $circle_id) {
```

Comment:
what will the error messages look like returned from the DB transaction blocks? If there's a validation error will we pass that to the FE or will it be hidden behind another transaction error log?

Reply:
The validation is performed before this code block in RequestForm, 
message: e.g {value} is required etc , if you need it changed then provide a list of error messages i'll update

Reply:
Let's merge for now and can update later.